### ISSUE TITLE
DynaSim error in Octave: regexp

### ISSUE THREAD

**Issue by kupiqu** [DynaSim/DynaSim#221]
```
dsDemos
error: regexp: range out of order in character class at position 47 of expression
error: called from
    dsClassifyEquation>classify at line 141 column 1
    dsClassifyEquation at line 76 column 13
    dsParseModelEquations at line 183 column 3
    dsCheckSpecification at line 350 column 6
    dsGenerateModel at line 155 column 14
    dsCheckModel at line 98 column 8
    dsSimulate at line 484 column 7
    dsDemos at line 49 column 5
```

**Comment by kupiqu** [comment]
Fixed, now dealing with:

```
dsDemos
Error: regexp: range out of order in character class at position 33 of expression
         in dsStrrep (line 48)
         in dsWriteDynaSimSolver (line 613)
         in dsGetSolveFile (line 151)
         in dsSimulate>tryFn (line 907)
         in dsSimulate (line 747)
         in dsDemos (line 49)
regexp: range out of order in character class at position 33 of expression
error: called from
    dsSimulate at line 767 column 5
    dsDemos at line 49 column 5
```

**Comment by kupiqu** [comment]
Fixed, now dealing with:
```
dsDemos
error: regexp: unmatched parentheses at position 0 of expression
error: called from
    dsParseModelEquations at line 287 column 13
    dsCheckSpecification at line 350 column 6
    dsGenerateModel at line 155 column 14
    dsCheckModel at line 98 column 8
    dsSimulate at line 484 column 7
    dsDemos at line 87 column 5
```

**Comment by kupiqu** [comment]
fixed in b980961
